src/en/administration/first_steps.md

```diff
@@ -0,0 +1,36 @@
+# Administration First Steps
+
+After you successfully installed Lemmy either [manually with Docker](https://join-lemmy.org/docs/en/administration/install_docker.html) or [automatically with Ansible](https://join-lemmy.org/docs/en/administration/install_ansible.html) here are some recommendations for a new administrator of a Lemmy server.
+
+## Admin Settings
+
+The first thing to do is to go to your admin panel, which can be found by clicking on the cog at the top right next to the search icon. Here you define a description for your site, so that people know if it is about one specific topic or if all subjects are welcome. You can also add an icon and a banner that define your server, it can for example be the logo of your organization.
+
+Take the time to browse through the entire page to discover the different options you have to customize your Lemmy instance, on the same page you can edit your [configuration file](https://join-lemmy.org/docs/en/administration/configuration.html), where you can find information about your database, the email used by the server, the federation options or the main administrator.
```

Comment:
Better use relative links (`administration/configuration.md` should work), then it also works with `mdbook serve` (or if someone wants to publish the docs on their own site).